lib/services.js

```diff
@@ -72,10 +74,19 @@ module.exports = function generateServices(app, options) {
 
 function describeModels(app, options) {
   var result = {};
+  var modelsToIgnore = options.modelsToIgnore;
+
   app.handler('rest').adapter.getClasses().forEach(function(c) {
     var name = c.name;
     c.description = c.sharedClass.ctor.settings.description;
 
+    if(modelsToIgnore.indexOf(name) >= 0){
+      // Skip classes that are provided in options.modelsToIgnore array
+      // We don't want to publish these models in angular app
+      console.error('Skipping %j model as it is not to be published', name);
```

Comment:
@bajtos and @superkhau Thanks for this. I will update it with `console.warn` or maybe `console.log`. I did this to highlight the log across all other logs. But definitely that's not the right way and I should change it.